src/workloads/sharding/WriteOneWithoutShardKeyShardedCollection.yml

```diff
@@ -0,0 +1,127 @@
+# TODO SERVER-3208: Enable Workload.
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On a sharded collection on a single shard cluster, run workloads that updateOne, deleteOne, and findAndModifyOne.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+
+  Database: &Database test
+  # Collection0 is the default collection populated by the MonotonicSingleLoader.
+  ShardedCollection: &ShardedCollection Collection0
+  Namespace: &Namespace test.Collection0
+
+  DocumentCount: &DocumentCount 10000  # Number of documents to insert and modify.
+  BalancerWait: &BalancerWait "20 seconds"  # Wait out the 20 second delay between balancing rounds
```

Comment:
I dont think this balancer wait is necessary since we dont have more than 1 shard now.